globus_automate_client/flows_client.py

```diff
@@ -478,49 +539,65 @@ def flow_action_log(
         :param reverse_order: An indicator for whether to retrieve the records
             in reverse-chronological order.
         """
-        if flow_scope is None:
-            flow_scope = self._scope_for_flow(flow_id)
-        self.authorizer = get_authorizer_for_scope(flow_scope, self.client_id)
+        if "authorizer" in kwargs:
```

Comment:
Seems that some of this is here because the `ActionClinet` is lacking the `log` method. That's a general Action method, so if it is not on the ActionClient it should move there and then this should follow the same pattern as the other Action related methods.

Reply:
`list-flow-actions` is also something we could move into the ActionClient (as the action-enumerate op) but I'm not sure it's time for that.

Reply:
Probably not time for that, but having the log in ActionClient is a good thing and I think I see that in the updated commit (again, will do a review on that separately).